content/en/docs/cheatsheets/troubleshooting.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Troubleshooting cheatsheet"
+linkTitle: "Troubleshooting"
+description: "Showcase various ways to get more information out of Flux controllers to debug potential problems."
+weight: 40
+---
+
+## Getting basic information
+
+Show all Flux objects that are not ready
+
+```cli
+flux get all -A --status-selector ready=false
+```
+
+Show flux warning events 
+
+```cli
+kubectl get events -n flux-system --field-selector type=Warning
+```
+
+Flux CLI (check for `Ready=True` and `Suspend=False`)
+
+```cli
+flux get sources all -A
+```
+
+See the CLI reference for [`get_sources_all`](/docs/cmd/flux_get_sources_all/).
+
+`kubectl` (check for `Ready=True`)
+
+```cli
+kubectl get gitrepositories.source.toolkit.fluxcd.io -A
+kubectl get helmrepositories.source.toolkit.fluxcd.io -A
+```
+
+Flux CLI (check for Ready=True and Suspend=False)
+
+```cli
+flux get kustomizations -A
+flux get helmreleases -A
+```
+
+CLI reference for [`get_kustomizations`](/docs/cmd/flux_get_kustomizations/) and [`get_helmreleases`](/docs/cmd/flux_get_helmreleases/).
+
+
+`kubectl` (check for `Ready=True`)
+
+```cli
+kubectl get kustomizations.kustomize.toolkit.fluxcd.io -A
+kubectl get helmreleases.helm.toolkit.fluxcd.io -A
+kubectl get helmcharts.source.toolkit.fluxcd.io -A
+```
+
+Looking for controller errors:
+
+```cli
+kubectl logs -n flux-system -l app=source-controller | grep error
```

Comment:
Please replace this with an example of `flux logs`

Reply:
yep, let's replace it with:
`flux logs --all-namespaces --level=error`